frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -100,20 +136,34 @@ async function deleteUserRoleAssignment(
                     },
                 }"
             >
-                <TabPanel header="Users">
+                <TabPanel
+                    header="Application admins"
+                    v-if="selectedApplicationShortDisplayText === FAM_APPLICATION_NAME"
```

Comment:
I wonder if we use selectedApplicationShortDisplayText, or we check the id `selectedApplication.value.application_id == FAM_APPLICATION_ID`, cause if we use the application to check in the above, we better keep the same way. Maybe we could change this variable to be selectedApplicationId

Reply:
I'll do the change!